app/src/main/res/values/strings.xml

```diff
@@ -214,6 +214,7 @@
     <string name="look_and_feel_theme_color_pink">Pink</string>
     <string name="look_and_feel_theme_color_purple">Purple</string>
     <string name="look_and_feel_theme_color_woodland">Woodland</string>
+    <string name="look_and_feel_swipe_between_posts">Swipe between posts</string>
```

Comment:
I don't love this settings label. "Allow swiping between posts" might be a bit better while still being reasonably concise.